### ISSUE TITLE
SwiftFormat does not handle nested enum in case

### ISSUE THREAD

**Issue by nevil** [nicklockwood/SwiftFormat#124]
Not the most beautiful style but nonetheless.
SwiftFormat currently does not handle this code:

```swift
switch x {
case y:
    enum Foo {
        case z
    }
    break
default: break
}
```

```
:> ~/source/SwiftFormat/CommandLineTool/swiftformat swift-format-bug.swift
running swiftformat...
warning: unexpected token } at 5:4 in /Development/git-repos/iOS2/swift-format-bug.swift
error: failed to format any files
```

I'm not familiar enough with the code yet so can't propose a patch...

**Comment by nicklockwood** [comment]
Thanks - I've found a fix. Will include it in the next release.

**Comment by nicklockwood** [comment]
Fixed in 0.28.2

**Comment by nevil** [comment]
@nicklockwood
Firstly, thanks for a great tool!

I confirmed that it works with our code.
Thanks for fixing.
